src/InstrumentationEngine.Lib/InstrumentationMethod.cpp

```diff
@@ -89,15 +89,22 @@ HRESULT MicrosoftInstrumentationEngine::CInstrumentationMethod::InitializeCore(
 
     WCHAR wszModuleFullPath[MAX_PATH];
     memset(wszModuleFullPath, 0, MAX_PATH);
```

Comment:
I think this needs to be wmemset; otherwise it will only zero out the first 260 bytes instead of 520 bytes: sizeof(WCHAR) == 2 * sizeof(CHAR)

Reply:
Spoke offline, the PAL doesn't support wmemset, will log an issue to track this.